.github/workflows/coverage-comment.yml

```
@@ -0,0 +1,239 @@
+name: Coverage Comment
+
+on:
+  pull_request:
+    branches: [ main, develop ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)